src/core/defaults.json

```diff
@@ -169,5 +169,6 @@
   "flash-validation-error.postcode-format": "Postcode was invalid",
   "flash-validation-error.password-format": "Password must be at least 8 characters and with at least one number, lower and upper case letter",
   "flash-validation-error.url-format": "URL was invalid",
-  "flash-validation-error.candidates-maxItems": "You can only choose at most three candidates"
+  "flash-validation-error.candidates-maxItems": "You can only choose at most three candidates",
+  "telegram.welcome-message.md": "*Hello*, I'm your Callout bot\\. Use me to list and answer callouts from [beabee](https://beabee.io/)\\."
```

Comment:
You can then remove this from here once the value is stored in `Content`

Reply:
But how can I define a default message if none has been set yet? Just in the transformer?

Reply:
Just like this?
```ts
  telegram: withValue<"telegram">({
    welcomeMessageMd: ["data", "*Hello*, I'm your Callout bot\\. Use me to list and answer callouts from [beabee](https://beabee.io/)\\."]
  })
  ```

Reply:
:+1: